backfill-cli/src/main/java/com/datastax/oss/cdc/backfill/importer/PulsarImporter.java

```diff
@@ -126,19 +139,27 @@ public ExitStatus importTable() {
                         // For now, will mimic the metric by using now()
                         // TODO: Disable the e2e latency metric if the records are emitted from cdc back-filling CLI
                         final long tsMicro = Instant.now().toEpochMilli() * 1000;
-                        futures.add(mutationSender.sendMutationAsync(createMutation(pkValues.toArray(), this.exportedTable.getCassandraTable(), tsMicro)));
+                        sendMutationAsync(createMutation(pkValues.toArray(), this.exportedTable.getCassandraTable(), tsMicro));
```

Comment:
we are not handling failures

In case of failures we must report the error and retry

One proposal:
- keep accumulating a list of Futures
- every 1000 futures, wait for the results (then clear the list), fail the procedure in case there is one failure

Reply:
Error handling is implemented here: https://github.com/datastax/cdc-apache-cassandra/pull/118/files#diff-19495dbde526d4197418c17b952a3d8eb743402e9e8c7c01aafb05d503e28610R189

it logs, and the futures keep progressing - does that address your comment?